src/pocketmine/Player.php

```diff
@@ -3590,6 +3591,25 @@ public function sendTranslation($message, array $parameters = []){
 		$this->dataPacket($pk);
 	}
 
+    public function sendWeather(){
+        $pk1 = new LevelEventPacket();
+        $pk2 = new LevelEventPacket();
+        if($this->level->getWeather() == Level::WEATHER_RAIN_THUNDER){
+            $pk1->evid = LevelEventPacket::EVENT_START_RAIN;
+            $pk2->evid = LevelEventPacket::EVENT_START_THUNDER;
+        }elseif($this->level->getWeather() == Level::WEATHER_RAIN){
+            $pk1->evid = LevelEventPacket::EVENT_START_RAIN;
+            $pk2->evid = LevelEventPacket::EVENT_STOP_THUNDER;
```

Comment:
will sending the stop thunder evid cause glitches if the thunder wasn't already thundering?